src/FrenetOptimalTrajectory/fot_wrapper.cpp

```diff
@@ -48,7 +48,7 @@ extern "C" {
         FrenetPath* best_frenet_path = fot.getBestPath();
 
         int success = 0;
-        if (!best_frenet_path->x.empty()){
+        if (best_frenet_path && !best_frenet_path->x.empty()){
```

Comment:
Please update main.cpp as well. It also invokes getBestPath().

Reply:
Done!